pkg/api/v6/api.go

```diff
@@ -54,6 +58,17 @@ type GitRemoteConfig struct {
 	Path   string `json:"path"`
 }
 
+func (c GitRemoteConfig) SafeURL() string {
```

Comment:
It's a shame to have to recapitulate this method here -- is there any way of changing the type def so it just uses git.Remote?

Reply:
I embedded the `git.Remote` struct, and added a JSON tag to the `URL` field. This should ensure two things:

1. The JSON representation of the structure is the same
2. The old `(GitRemoteConfig{}).Remote.URL` should continue to work for whoever depends on it.